wolfssl/wolfcrypt/types.h

```diff
@@ -68,18 +68,22 @@
 	    #define WORD64_AVAILABLE
 	    #define W64LIT(x) x##ui64
 	    typedef unsigned __int64 word64;
+			typedef __int64 signed_word64;
```

Comment:
Our coding standard specified that we only use spaces (not hard tabs).  It looks like this line includes hard tabs.  Can you remove those?